client/src/main.c

```diff
@@ -88,7 +88,7 @@ struct AppParams params = { 0 };
 static void setGrab(bool enable);
 static void setGrabQuiet(bool enable);
 
-static void handleMouseGrabbed(double ex, double ey);
+void handleMouseGrabbed(double ex, double ey);
```

Comment:
While I can understand what you're doing here I am not sure it's the best way to do this. I am OK with this for now, but I may make an agnostic interface for this to make this easier for both X and Wayland code later.